### PR TITLE
foundry tests - ipToken

### PR DESCRIPTION
**Pull request by amydlarz**

### CODE REVIEW

File: test/tokens/IpToken.t.sol [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        // then
+        vm.prank(_admin);
+        vm.expectRevert(abi.encodePacked("Ownable: caller is not the owner"));
+        ipToken.transferOwnership(_user2);
+    }
+
+    function testShouldHaveRightsToTransferOwnershipSenderStillHaveRights() public {
+        // given
+        IpToken ipToken = prepareIpToken();
+
```
Comment (mario-ipor) [inline test/tokens/IpToken.t.sol:98]: Missing line in given section: ipToken.transferOwnership(_user1);

**Review by mario-ipor — CHANGES_REQUESTED** [review]

File: test/tokens/IpToken.t.sol (hunk 1 above)
Comment (amydlarz) [reply test/tokens/IpToken.t.sol:98]: it looks like a mistake in the hardhat test but I can do it like before

Comment (mario-ipor) [reply test/tokens/IpToken.t.sol:100]: transferOwnership

Comment (mario-ipor) [reply test/tokens/IpToken.t.sol:100]: No, it in on purpose, Ownable.transferOwnership from OpenZeppelin immediatelly change owner, here we wanted to show that went you executed transferOwnership first time, then you have rights to execute it again

**Review by mario-ipor — APPROVED** [review]

**Pull request merged**